cppcache/src/CacheImpl.cpp

```diff
@@ -814,14 +814,19 @@ DataInput CacheImpl::createDataInput(const uint8_t* buffer, size_t len,
 }
 
 PdxInstanceFactory CacheImpl::createPdxInstanceFactory(
-    const std::string& className) const {
+    const std::string& className, bool expectDomainClass) const {
   this->throwIfClosed();
 
   return PdxInstanceFactory(
-      className, *m_cacheStats, *m_pdxTypeRegistry, *this,
+      className, expectDomainClass, *m_cacheStats, *m_pdxTypeRegistry, *this,
       m_distributedSystem.getSystemProperties().getEnableTimeStatistics());
 }
 
+PdxInstanceFactory CacheImpl::createPdxInstanceFactory(
```

Comment:
I was thinking on removing this overload as this is not part of the public API what do you think?

Reply:
If it isn't publicly exported yes, remove extra cruft.